shells/firefox/panel/run.js

```diff
@@ -21,10 +23,21 @@ var node = document.getElementById('container');
 ReactDOM.render(<h2 id="message">Looking for React...</h2>, node);
 var port = {};
 
-// TODO (bvaughn) Read default :themeName and switch between 'FirefoxLight' and 'FirefoxDark'
-// Refer to 'shells/chrome/src/panel.js' for an example
 const browserName = 'Firefox';
-const themeName = 'FirefoxLight';
+let themeName = 'FirefoxLight';
+
+// If available, mirror the current Firefox devtools theme:
+// https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/devtools.panels/themeName
+if (typeof devtools !== 'undefined' && devtools.panels) {
+  switch (devtools.panels.themeName) {
```

Comment:
Ah! Figured it out. Seems like I need to mimic the structure in shells/chrome for firefox, and then run [`web-ext run --firefox=/Applications/FirefoxNightly.app/Contents/MacOS/firefox-bin`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext) from the directory.